lametro/views.py

```diff
@@ -238,33 +240,31 @@ def get_context_data(self, **kwargs):
 
 
 def handle_uploaded_agenda(agenda, event):
-    with open('lametro/static/pdf/%s' % agenda.name, 'wb+') as destination:
+    with open('lametro/static/pdf/agenda-%s.pdf' % event.slug, 'wb+') as destination:
         for chunk in agenda.chunks():
             destination.write(chunk)
 
     # Create the document in database
     document_obj, created = EventDocument.objects.get_or_create(event=event,
-        url='pdf/%s' % agenda.name, updated_at= timezone.now())
+        url='pdf/agenda-%s.pdf' % event.slug, updated_at= timezone.now())
     document_obj.note = ('Event Document - Manual upload PDF')
     document_obj.save()
 
-
-# from fabric.api import *
-# # Hosts to deploy onto
-# # env.hosts = ['www1.example.com', 'www2.example.com']
-
-# # Where your project code lives on the server
-# env.project_root = '/home/datamade/lametro-staging'
-
-# def deploy_static():
-#     with cd(env.project_root):
-#         run('./manage.py collectstatic -v0 --noinput')
+    # Collect static to render PDF on server
+    management.call_command('collectstatic', '--noinput')
 
 
 def delete_submission(request, event_slug):
     event = Event.objects.get(slug=event_slug)
     event_doc = EventDocument.objects.filter(event_id=event.ocd_id, note__icontains='Manual upload')
-    for e in event_doc: 
+
+    for e in event_doc:
+        # Remove stored PDF from Metro app.
+        if 'Manual upload PDF' in e.note:
+            try:
+                os.remove('lametro/static/%s' % e.url )
+            except OSError:
+                pass
```

Comment:
Is there any reason we might want to log this?

Reply:
No - indeed, I am being a little overcautious here, since a PDF should not reside in the static files, if a corresponding entry for an EventDocument does not exist in the database. But...just in case a PDF disappears or somehow does not get created, then the Metro admin will not see an error when they try to delete a mis-uploaded agenda.